common-arcana.lic

```diff
@@ -236,4 +236,57 @@ module DRCA
       return parse_mana_message(mana_msg)
     end
   end
+
+  def check_osrel(osrel_no_harness, osrel_amount)
+    return unless DRSpells.active_spells['Osrel Meraud'] && DRSpells.active_spells['Osrel Meraud'] < 90
+    infuse_om(!osrel_no_harness, osrel_amount)
+  end
+
+  def do_waggle_set(settings, spells)
+    check_osrel(settings.osrel_no_harness, settings.osrel_amount)
```

Comment:
Do we want to `check_osrel` before casting the entire set, or before casting each spell in the set?